@here/harp-mapview/lib/Utils.ts

```diff
@@ -450,7 +444,7 @@ export namespace MapViewUtils {
             mapView.projection.type === ProjectionType.Spherical
                 ? cache.vector3[0].copy(mapView.camera.position).normalize()
                 : cache.vector3[0].set(0, 0, 1),
-            MathUtils.degToRad(-deltaYawDeg)
+            THREE.Math.degToRad(-deltaYawDeg)
```

Comment:
`THREE.Math` got renamed to `THREE.MathUtils` with the three 0.113 update I just merged

Reply:
Ok will change it.